chroot-bin/xiwi

```diff
@@ -7,17 +7,21 @@
 
 USAGE="Usage: ${0##*/} [-f] [-F|-T] APPLICATION [PARAMETERS ...]
 Launches a windowed session in Chromium OS for any graphical application.
+Applications launched in this way show in independent windows or tabs.
 All parameters are passed to the specified application.
 
-By default, the app is launched in a window. Passing -F will launch the app
```

Comment:
Your right, sorry, I'll put it back in.
I guess I glossed over the `By default, the app is launched in a window.` part.